core/chains/evm/txmgr/eth_broadcaster.go

```diff
@@ -499,6 +516,12 @@ func (eb *EthBroadcaster) handleInProgressEthTx(ctx context.Context, etx EthTx,
 	}
 
 	if sendError == nil {
+		// We want to observe the time until the first _successful_ broadcast.
+		// Since we can re-enter this method by way of tryAgainBumpingGas,
+		// and we pass the same initialBroadcastAt timestamp there, when we re-enter
```

Comment:
Perhaps a poorly worded comment - but I _want_ to pass `time.Now()` because `initialBroadcastAt` is unchanged even after we try bumping gas. The call graph looks like `handleInProgressEthTx -> tryAgainBumpingGas -> tryAgainBumpingLegacyGas/tryAgainBumpingDynamicFeeGas -> tryAgainWithNewLegacyGas/tryAgainWithNewDynamicFeeGas -> handleInProgressEthTx`. I guess the time difference won't be that big, unless we do a backoff?